config/policy.csv

```diff
@@ -1 +1,12 @@
-p, /validate, GET
+# admin have all priviliges
+p, admin, *, *
+
+# anyone could perform read-only operation
+p, *, *, get
+p, *, *, list
+
+# member could perform write operations
+p, member, /validate, create
```

Comment:
@hsluoyz , can the obj in the policy file be something else other than the url path?
The above policy file will authorize based on the act (operation). Like the user having the role of member could perform the assigned operation on any resources. 

But is there a way  to restrict it? 

`p, member, cluster, create`  
`p, member, pod, create`

Like could the above lines work? if the obj is not an urlpath?

Reply:
Since the phase-1 is over, I wrote an article on medium sharing my experience and a detailed description of the project.
https://medium.com/@malikashish493/gsoc21-phase-1-casbin-88f69f7b07fe

Reply:
@ashish493 obj can be any string. It can be URL path, or an object name like book1, book2, or even an ID like 464355762.